rskj-core/src/test/java/co/rsk/core/types/bytes/BytesSliceTest.java

```diff
@@ -74,4 +113,44 @@ void testEmptySlice() {
         assertEquals(0, actualResult.length());
         assertArrayEquals(expectedResult, actualResult.copyArray());
     }
+
```

Comment:
suggestion:

It's kinda hard to understand straight away what it's going on here. I suggest at least add a comment above this function to understand what is each object of this function.

```suggestion
//      Inputs for the function, in order: src, srcPos, dest, destPos, length. 
```